src/ClientData/include/ClientData/ModuleData.h

```diff
@@ -48,30 +48,29 @@ class ModuleData final {
   [[nodiscard]] bool UpdateIfChangedAndNotLoaded(orbit_grpc_protos::ModuleInfo info);
   // offset here is the absolute address minus the address this module was loaded at by
   // the process (module base address)
-  [[nodiscard]] const orbit_client_protos::FunctionInfo* FindFunctionByOffset(uint64_t offset,
-                                                                              bool is_exact) const;
-  [[nodiscard]] const orbit_client_protos::FunctionInfo* FindFunctionByElfAddress(
-      uint64_t elf_address, bool is_exact) const;
+  [[nodiscard]] const FunctionInfo* FindFunctionByOffset(uint64_t offset, bool is_exact) const;
+  [[nodiscard]] const FunctionInfo* FindFunctionByElfAddress(uint64_t elf_address,
+                                                             bool is_exact) const;
   void AddSymbols(const orbit_grpc_protos::ModuleSymbols& module_symbols);
-  [[nodiscard]] const orbit_client_protos::FunctionInfo* FindFunctionFromHash(uint64_t hash) const;
-  [[nodiscard]] const orbit_client_protos::FunctionInfo* FindFunctionFromPrettyName(
-      std::string_view pretty_name) const;
-  [[nodiscard]] std::vector<const orbit_client_protos::FunctionInfo*> GetFunctions() const;
+  [[nodiscard]] const FunctionInfo* FindFunctionFromHash(uint64_t hash) const;
+  [[nodiscard]] const FunctionInfo* FindFunctionFromPrettyName(std::string_view pretty_name) const;
+  [[nodiscard]] std::vector<const FunctionInfo*> GetFunctions() const;
+
+  static std::string GetLoadedModuleNameByPath(std::string_view module_path);
```

Comment:
As said in the other comment. Seems odd to have this here, since its just a std::filesystem function and has nothing to do with ModuleData.

Reply:
I removed that function.